Laboratoare/Lab/src/polimorfism/birdcontrollerex1/hierarchyex5/Shoes.java

```diff
@@ -1,27 +0,0 @@
-package polimorfism.birdcontrollerex1.hierarchyex5;
-
-enum Brand{
```

Comment:
enumurile se pun de obicei in fisiere diferite, gen faci un fisier Brand unde pui enumul Brand, un fisier pentru Size si tot asa haw haw